### PR TITLE
stow: add page

### PR DESCRIPTION
**Pull request by mstruebing**
Gnu Stow - A symlink manager.

More information here: https://www.gnu.org/software/stow/

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/stow.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# stow
+
+> Symlink manager.
+> Often used to manage dotfiles.
+
+- Symlink all files recursively to {{target}}:
```
Comment (agnivade) [inline pages/common/stow.md]: From the docs, I see that target is a directory right ? So shall we change target to "path/to/directory" ?

File: pages/common/stow.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`stow --target={{target}} {{file1 folder1 file2 folder2}}`
+
+- Delete symlinks recursively from {{target}}:
+
+`stow --delete --target={{target}} {{file1 folder1 file2 folder2}}`
+
+- Simulate to see what the result would be like:
+
+`stow  --simulate --target={{target}} {{file1 folder1 file2 folder2}}`
```
Comment (agnivade) [inline pages/common/stow.md]: multiple spaces between stow and --simulate

File: pages/common/stow.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Delete and resymlink:
+
+`stow --restow --target={{target}} {{file1 folder1 file2 folder2}}`
+
+- Exclude files matching a regular expression:
+
+`stow  --ignore={{regex}} --target={{target}} {{file1 folder1 file2 folder2}}`
```
Comment (agnivade) [inline pages/common/stow.md]: multiple spaces.

File: pages/common/stow.md (hunk 1 above)
Comment (mstruebing) [reply pages/common/stow.md]: Yep, thx!

File: pages/common/stow.md (hunk 2 above)
Comment (mstruebing) [reply pages/common/stow.md]: Thx for your catchy eye! :)

**Comment by mstruebing** [comment]
updated

File: pages/common/stow.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# stow
+
+> Symlink manager.
+> Often used to manage dotfiles.
+
+- Symlink all files recursively to {{/path/to/target/directory}}:
```
Comment (pxgamer) [inline pages/common/stow.md]: Please could you remove the starting `/` from the path here?

File: pages/common/stow.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`stow --target={{directory}} {{file1 folder1 file2 folder2}}`
+
+- Delete symlinks recursively from {{/path/to/target/directory}}:
```
Comment (pxgamer) [inline pages/common/stow.md]: And here. Cheers. 👍

File: pages/common/stow.md (hunk 4 above)
Comment (pxgamer) [reply pages/common/stow.md]: Oh, actually, maybe it should also be `path/to/target_directory` or just `path/to/directory`. Up to you 😄 👍

**Comment by mstruebing** [comment]
Done.

**Comment by mstruebing** [comment]
Yep it's okay so! :)

**Comment by mstruebing** [comment]
Thank you for correcting :)

**Pull request merged**